pkg/vet/api_resources.go

```diff
@@ -91,8 +91,9 @@ func addLine(scoper *discovery.Scoper, path cmpath.Absolute, line string) status
 	kind := fields[len(fields)-1]
 	namespaced := fields[len(fields)-2]
 	group := ""
-	if len(fields) > 2 {
-		group = fields[0]
+	if len(fields) > 2 && len(fields[0]) > 2 {
```

Comment:
What's the `len(fields[0]) > 2` checking?

Reply:
as described in https://github.com/GoogleContainerTools/kpt-config-sync/pull/868/files#diff-d95c9ef5c2ae2bcee59a43bcbb92b0026aef90af3f2b589d71dd883079787699R83, fields is APIVERSION, NAMESPACED, and KIND. `len(fields[0]) > 2` is intended to check whether APIVERSION is just `v1` or contains group like `apps/v1`. I'll update the code to be more clear and add comments to explain